number/src/goldilocks.rs

```diff
@@ -0,0 +1,26 @@
+use ark_ff::{Fp64, MontBackend, MontConfig};
+
+#[derive(MontConfig)]
+#[modulus = "18446744069414584321"]
+#[generator = "7"]
+pub struct GoldilocksBaseFieldConfig;
+pub type GoldilocksBaseField = Fp64<MontBackend<GoldilocksBaseFieldConfig, 1>>;
```

Comment:
Does ark not have GL?

Reply:
Nein https://github.com/arkworks-rs/curves
Also note that even the curves ark "has" just use this macro under the hood for the field.